src/i18n/locales/en/translations.json

```diff
@@ -484,10 +484,10 @@
   "wallet.assets.header.value": "{{num}}<0>{{denom}}</0>",
   "wallet.assets.table.title": "Available assets",
   "wallet.assets.table.toggle": "Show all assets",
-  "wallet.assets.table.addToken": "Add New Asset",
-  "wallet.assets.table.updateToken": "Review Changes",
-  "wallet.assets.table.addToken.unknown": "This is unknown asset. Add it to your wallet to see details.",
-  "wallet.assets.table.addToken.changed": "This asset details has changed. Please review for your safety.",
+  "wallet.assets.table.addToken": "Add a new asset",
```

Comment:
Add new asset (would be an but not needed here)

Reply:
There is still extra a

Reply:
fixed